source/jormungandr/jormungandr/interfaces/v1/ResourceUri.py

```diff
@@ -173,31 +173,35 @@ def wrapper(*args, **kwargs):
                 return data
         return wrapper
 
+
 class complete_links(object):
     def __init__(self, resource):
         self.resource = resource
-    def complete(self, data, collect):
+        self.excepted_keys = set(['category', 'id', 'internal', 'rel', 'type'])
+
+    def make_and_get_link(self, elem, collect):
+        if collect == "notes":
+            return {"id": elem['id'], "category": elem['category'], "value": elem['value'], "type": collect}
```

Comment:
I'm not sure : are the tests OK if you remove category here and in excepted_keys?
They should be, but if you didn't try it's worth the test (even if we can't let it in the code)